generators/app/package.json

```diff
@@ -2,18 +2,22 @@
   "name": "polymer-starter-kit-custom-build",
   "version": "0.0.0",
   "description": "A starting point for Polymer 1.0 custom build apps",
-  "license": "http://polymer.github.io/LICENSE.txt",
+  "license": "BSD-3-Clause",
   "scripts": {
-    "build": "gulp"
+    "lint": "eslint . --ext js,html; exit 0;",
```

Comment:
I think I had this in PSK because I wanted a clear separation between when linting was finished and when the tests began. Can't recall now :P But I'm ok removing it.